lib/ui/views/profile/user_favourites_view.dart

```diff
@@ -47,6 +49,36 @@ class _UserFavouritesViewState extends State<UserFavouritesView>
           });
         }
 
+        if (_items.isEmpty == true) {
+          return Center(
+              child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Text(
+                'No Favourites :(',
+                style: TextStyle(fontSize: 25, color: Colors.white),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(top: 20),
+              ),
+              FlatButton(
+                color: CVTheme.primaryColor,
+                child: Text(
+                  'Explore circuits',
+                  style: TextStyle(fontSize: 25, color: Colors.white),
+                ),
+                onPressed: () {
+                  Navigator.push(
+                      context,
+                      MaterialPageRoute(
+                          builder: (BuildContext context) =>
+                              FeaturedProjectsView()));
```

Comment:
Use trailing commas in the end, https://flutter.dev/docs/development/tools/formatting#using-trailing-commas